src/IBM.Watson.TextToSpeech.v1/Websockets/MarkTiming.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace IBM.Watson.TextToSpeech.v1.Websockets
+{
+    public class MarkTiming
+    {
+        public string word { get; }
```

Comment:
use 

```cs
[JsonProperty("word", NullValueHandling = NullValueHandling.Ignore)]
public string Word { get; }
```

Reply:
All props should be PascalCase (UpperCamelCase). Use JsonProperty to add the real prop name.